script/vsts/lint.yml

```diff
@@ -0,0 +1,12 @@
+jobs:
+  - job: Lint
+    timeoutInMinutes: 10
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - script: |
+          cd script
+          npm ci
```

Comment:
```suggestion
          npm install -g @aminya/group-dependencies
          cd script
          deps install lint
```

Reply:
I think I'd rather leave that to a followup PR. The separation of the lint job works without needing any extra dependencies (I also just dislike adding dependencies, so don't want to make that PR myself).

Reply:
OK, I will create a separate PR.